packages/next/build/index.ts

```diff
@@ -145,6 +156,8 @@ export default async function build(dir: string, conf = null): Promise<void> {
     result = await runCompiler(configs)
   }
 
+  const buildEnd = process.hrtime(buildStart)
```

Comment:
This only covers the webpack part of next build, would probably be good to name it as such I guess?